Create new miniforge Blogpost (#80)
BiAPoL/blog: docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md

> # Getting started with Miniforge* and Python 

Reviewer: ```suggestion
# Getting started with Miniforge and Python 
```

[Mara Lampert](../readme), July 8th, 2024

> > **_Note:_** `(*)` An older version of this blog post covered Mambaforge. After the release of Miniforge 23.3.1 in August 2023, Miniforge and Mambaforge are essentially identical. The only difference is the name of the installer and subsequently the default installation path. In the current version of this blog post, we describe Miniforge3.

Reviewer: ```suggestion
```

## Introduction to Python and Miniforge 
This blog post explains what Python and [Conda](https://docs.readthedocs.io/en/stable/guides/conda.html) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 

> > **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/Readme.html) written by [Johannes](https://biapol.github.io/blog/johannes_mueller/Readme.html). 

Reviewer: ```suggestion
> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html). 
```

**Why do we need Conda to use Python?**

__Python__ is a programming language which is easy to learn and very important in scientific data analysis nowadays. 

__Conda__ is a __package manager__ which can be used with Python. It is a software allowing to install other software. Read more about Conda [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). 

## Installation of Miniforge 

> Here, I am going to show how to install Miniforge. It comes with everything you need and downloads it from a community-driven open source software provider called  [conda-forge](https://conda-forge.org/). 

Reviewer: ```suggestion
Here, I am going to show how to install Miniforge. It comes with everything you need and downloads packages from a community-driven open source software provider called  [conda-forge](https://conda-forge.org/). 
```
